web

```diff
@@ -1 +0,0 @@
-Subproject commit 83463c2bcd5c4360adbc41c61e6572406fec54ce
```

Comment:
This seems like a weird change to have in this PR. IIRC, we stopped pulling latest because they added features that expected different things from the bot. But I'm not the most knowledgable person on this.